### PR TITLE
Add TIL about using HTML in i18n locale files

### PR DESCRIPTION
**Pull request by scottmatthewman**

### CODE REVIEW

File: activesupport/html_in_i18n.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+```yml
+# locales/en.yml
+header:
+  tagline: "The best website for <strong>%{subject}</strong>"
+```
+
+However, this has the potential to allow `subject` to include rogue HTML (or worse, Javascript XSS vulnerabilities) which could then be rendered in the browser.
+
+An alternative is to name the key with an `_html` prefix. This allows us to use HTML elements within the text of our YAML locale file, but any parameters that are supplied are escaped. So if any of the data we are passing in has been supplied by users, they should still be safe to display.
```
Comment (ascastro) [inline activesupport/html_in_i18n.md]: It should be `suffix` instead of `prefix`, right?

Comment (scottmatthewman) [inline activesupport/html_in_i18n.md]: Just testing :blush:

**Comment by ascastro** [comment]
I've learnt something too! :)

File: activesupport/html_in_i18n.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+header:
+  tagline: "The best website for <strong>%{subject}</strong>"
+```
+
+However, this has the potential to allow `subject` to include rogue HTML (or worse, Javascript XSS vulnerabilities) which could then be rendered in the browser.
+
+An alternative is to name the key with an `_html` suffix. This allows us to use HTML elements within the text of our YAML locale file, but any parameters that are supplied are escaped. So if any of the data we are passing in has been supplied by users, they should still be safe to display.
+
+```haml
+# views/page2.html
```
Comment (jbilbo) [inline activesupport/html_in_i18n.md:24]: should this be `# views/page2.html.haml`? 
Also, would it make sense to put the examples in the same order as before? First the `yml` and then the `html`?

**Pull request merged**